mentorship ios/Constants/LocalizableStringConstants.swift

```diff
@@ -32,8 +32,10 @@ struct LocalizableStringConstants {
     static let failure = LocalizedStringKey("Failure")
     static let profile = LocalizedStringKey("Profile")
     static let editProfile = LocalizedStringKey("Edit Profile")
+    static let addTask = LocalizedStringKey("Add Task")
     static let relationRequest = LocalizedStringKey("Relation Request")
     static let notAvailable = LocalizedStringKey("Not available")
+    static let operationFail = LocalizedStringKey("Operation failed. Please try again")
```

Comment:
Say something like "Task not added"? "Operation failed" seems like a really serious message?

Reply:
Done